src/Bumblebee.IntegrationTests/Extensions/VerificationTests.cs

```diff
@@ -99,15 +124,144 @@ public void given_selection_verification_passes_when_verifying_selected_should_r
 
 		[TestCase(true)]
 		[TestCase(false)]
-		public void given_selection_verification_fails_when_verifying_selected_should_throw_exception(bool expected)
+		public void Given_selection_verification_fails_when_verifying_selected_should_throw_exception(bool expected)
 		{
 			var element = Substitute.For<ISelectable>();
 			element.Selected.Returns(!expected);
 
 			Action action = () => element.VerifySelected(expected);
+
 			action
 				.ShouldThrow<VerificationException>()
-				.WithMessage("Selection verification failed. Expected: {0}, Actual: {1}.".FormatWith(expected, element.Selected));
+				.WithMessage(String.Format("Selection verification failed. Expected: {0}, Actual: {1}.", expected, element.Selected));
+		}
+
+		[Test]
+		public void Given_verification_on_IClickable_When_verification_fails_and_take_screenshot_is_true_Then_screenshot_is_taken()
```

Comment:
The "take_screenshot_is_true" is part of the Given (context) of the text rather than the action (when clause).  I will make this change - just capturing this.